code/unclassified/src/maximum_subarray_sum/maxsubarraysum.cpp

```diff
@@ -0,0 +1,35 @@
+//Part of Cosmos by OpenGenus Foundation
+
+
+// C++ implementation of simple algorithm to find
+// Maximum Subarray Sum in a given array
+//this implementation is done using Kadane's Algorithm which has a time complexity of O(n)
+
+#include<bits/stdc++.h>
```

Comment:
#1915. Don't use using namespace std, and use `std::vector` instead of dynamic arrays.

Reply:
done